src/components/NewSeedBank/index.tsx

```diff
@@ -209,6 +239,67 @@ export default function SeedBankView(): JSX.Element {
                 tooltip={strings.TOOLTIP_TIME_ZONE_SEEDBANK}
               />
             </Grid>
+            <Grid item xs={gridSize()}>
+              <DatePicker
+                id={'buildStartedDate'}
+                label={strings.FACILITY_BUILD_START_DATE}
```

Comment:
use labels for Seed Bank instead of Nursery?

Reply:
Changed the name of these strings to "Facility"

Reply:
Clever :)

Reply:
Oops but i see that the word Nursery is in the string. will fix

Reply:
Looks good! Removing any mention of nursery or seed bank is smipler.